root/layout/components/MetaDescription.js

```diff
@@ -20,18 +22,24 @@ function entityDescription(entity) {
   if (entity.comment) {
     desc.push(entity.comment);
   }
-  if (entity.type) {
-    desc.push(l('Type:') + ' ' + entity.type.name);
-  }
   return desc;
 }
 
+function pushTypeName(desc, entity) {
+  const typeName = entity.typeName;
+  if (typeName) {
+    desc.push(l('Type:') + ' ' + typeName);
```

Comment:
This info is all stuffed into a `<meta name="description" />` tag which search engines can use to display in search results. When they crawl the site everything will be in English anyway, so I'm not sure why we have `l('Type:')` etc. here actually.

Reply:
In that case, maybe we should remove those - it'd just feel more reasonable to have both work the same way :)

Reply:
For Google at least, https://support.google.com/webmasters/answer/182192?hl=en&ref_topic=2370587 is informative. The translations would be useful if we had separate URLs for each language, but per the above link, "[if] you prefer to dynamically change content or reroute the user based on language settings, be aware that Google might not find and crawl all your variations."

Reply:
Hmm. So, unless we make something like fr.musicbrainz.org, Google is unlikely to ever see these? Well, then I'd say either decide there's no point and remove the l(), or decide we're ok with the small likelihood and add the other l() :)

Reply:
I guess I'd lean towards removing it, since even now only the labels are translated, not the actual content, and putting work into making it fully translated seems pointless if Google isn't likely to use it.